components/instructions/programs/lifinity.tsx

```diff
@@ -74,9 +77,12 @@ export const LIFINITY_PROGRAM_INSTRUCTIONS = {
           .shiftedBy(-tokenLPInfo.account.decimals)
           .toString();
 
+        const poolLabel = getPoolLabelByPoolMint(tokenLPAcc.toBase58());
+
         return (
           <>
-            <p>{`Max Amount of Token A to deposit: ${uiAmountTokenA}`}</p>
+            <p>{`Liquidity Pool: ${poolLabel}`}</p>
```

Comment:
nit: 

We gotta set a standard about how to display data on `instructions/program`  :)